docs/advanced-features/workflows.md

```diff
@@ -423,5 +423,6 @@ and
 
 ## Known limitations
 * Only [Draft 7 Json Schemas](https://json-schema.org/specification-links.html#draft-7) are supported
+  * If a workflow schema includes a non-supported keyword, the user interface displays an `unknown keyword: <non-supported keyword>` error
```

Comment:
This line is longer than 80 symbols (as line above and beneath)

Reply:
Fixed in latest commit